ecosystem/platform/server/app/views/settings/connections.html.erb

```diff
@@ -0,0 +1,31 @@
+<% content_for(:page_title, 'Connection Settings') %>
+<h3 class="text-4xl mb-8 font-display font-light">Connections</h3>
+<% User.omniauth_providers.each do |provider| %>
+  <div class="mb-8 flex justify-between">
+    <div class="flex gap-4 items-center">
+      <%= render IconComponent.new(provider, size: :large) %>
+      <div>
+        <h4 class="text-xl font-display font-light"><%= provider.to_s.titleize %></h4>
+        <% if @authorizations[provider.to_s] %>
+          <p><%= @authorizations[provider.to_s].email || @authorizations[provider.to_s].username %></p>
```

Comment:
Can we make a method on authorizations to specify the display value? i.e email for gmail, username for discord + github, etc

Reply:
Done.